### ISSUE TITLE
Screen reader doesn't announce first result

### ISSUE THREAD

**Issue by shaunbarrio** [trevoreyre/autocomplete#103]
Hi @trevoreyre - great component, it works really well so thanks for sharing.

I do have one small issue using the JavaScript version. When using VoiceOver (on Chrome MacOS), the first result isn't announced when first moving to the results using the down arrow key. However, all other results are announced as expected. The first result is then announced if you navigate to it a second time using the arrow keys but this isn't ideal.

Just to confirm, this is with or without `autoSelect: true` and also occurs in the demos. Please let me know if there's something that can be done to resolve this, I haven't had any success in my attempts to do so using the available options.

Thanks

**Comment by trevoreyre** [comment]
Thanks, @shaunbarrio. I'll have to investigate further on this one. If it's not announcing the first result until you navigate to it the second time, that's definitely a problem.

**Comment by shaunbarrio** [comment]
Hi @trevoreyre, I hope you're well.

I was just wondering if you were ever able to investigate this screen reader issue further?

Thanks
